dts/msm8916/rules.mk

```diff
@@ -1,5 +1,9 @@
 LOCAL_DIR := $(GET_LOCAL_DIR)
 
+# To build for htc-m8qlul, comment out all dtbs and add
+# $(LOCAL_DIR)/msm8939-htc-m8qlul.dtb
+# m8qlul does not work with all dtbs enabled.
```

Comment:
Can you put the comment in the dts file (and adjust reference in README). Seems a bit better there.